daemon/src/model/cfd.rs

```diff
@@ -134,27 +134,29 @@ pub struct CfdStateCommon {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 #[serde(tag = "type", content = "payload")]
 pub enum CfdState {
-    /// The taker has requested to take a CFD, but has not messaged the maker yet.
+    /// The taker sent an order to the maker to open the CFD but doesn't have a response yet.
     ///
-    /// This state only applies to the taker.
-    TakeRequested {
+    /// This state applies to taker only.
+    OutgoingOrderRequest {
         common: CfdStateCommon,
     },
-    /// The taker sent an open request to the maker to open the CFD but don't have a response yet.
+
+    /// The maker received an order from the taker to open the CFD but doesn't have a response yet.
     ///
-    /// This state applies to taker and maker.
-    /// Initial state for the maker.
-    PendingTakeRequest {
+    /// This state applies to the maker only.
+    IncomingOrderRequest {
         common: CfdStateCommon,
+        taker_id: TakerId,
     },
```

Comment:
This is where we might want to split the states into one per party. @da-kami what do you think?